recipes/libcurl/all/conanfile.py

```diff
@@ -1,15 +1,18 @@
-import glob
-import os
-import re
-import functools
-from conans import ConanFile, AutoToolsBuildEnvironment, CMake, tools
+from conan import ConanFile
 from conan.errors import ConanInvalidConfiguration
-from conan.tools.microsoft import is_msvc
+from conan.tools.apple import is_apple_os, fix_apple_shared_install_name
 from conan.tools.build import cross_building
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.env import VirtualBuildEnv, VirtualRunEnv
+from conan.tools.files import apply_conandata_patches, copy, download, get, load, replace_in_file, rm, rmdir, save
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.microsoft import is_msvc, unix_path
 from conan.tools.scm import Version
-from conan.tools.files import replace_in_file, rmdir, chdir, get, download, save, load, apply_conandata_patches
-from conan.tools.apple import is_apple_os
+# TODO: Migrate
+from conans.tools import get_env
```

Comment:
This has been dropped in other PRs... not sure thats the right thing or not

Reply:
Indeed: here is an example 33c236a4a8c6d0fd95f2c27dfda13db7f2474052

I dropped it here: 9f24973c0